packages/masternode-reward-shares-contract/README.md

```diff
@@ -0,0 +1,33 @@
+# Reward sharing Contract
+
+[![Build Status](https://github.com/dashevo/dpns-contract/actions/workflows/test_and_release.yml/badge.svg)](https://github.com/dashevo/dpns-contract/actions/workflows/test_and_release.yml)
```

Comment:
Just noticed the badges, description, and install command all refer to the DPNS contract, not reward sharing.

Reply:
Too late :( could you please create one more PR to fix it?

Reply:
Actually it appears nothing is published on npm so they would just be broken links. Probably should update at release? This also reveals the larger issue that all build status and release badges in this repo need to be updated.